ntbs-service/Pages/Shared/_Header.cshtml

```diff
@@ -0,0 +1,46 @@
+<header class="govuk-header " role="banner" data-module="govuk-header">
+    <div class="header-title-and-logo-background">
+        <div class="govuk-header__container govuk-width-container no-border">
+            <button type="button" role="button" class="govuk-header__menu-button govuk-js-header-toggle header-expand-menu-button" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+
+            <div class="govuk-header__logo header-logo-container header-logo">
+            <a href="#" class="govuk-header__link govuk-header__link--homepage">
+                <span class="govuk-header__logotype">
+                    <img src="~/Images/pheLogo.png" class="header-phe-logo"></img>
+                </span>
+            </a>
+        </div>
+        <a href="#" class="govuk-header__link govuk-header__link--service-name header-title">

```

Comment:
ditto - I imagine we just don't want this to be a link, since we have a dedicated "Home" one below?